numpy/linalg/tests/test_linalg.py

```diff
@@ -729,7 +831,10 @@ def test_square(self):
 
 class TestEigvalsh(HermitianTestCase, HermitianGeneralizedTestCase):
 
-    def do(self, a, b):
+    def do(self, a, b, flags):
+        if flags & CaseFlags.empty:
+            assert_raises(LinAlgError, linalg.eigvalsh, a, 'L')
```

Comment:
Darn, It turns out `empty` is a confusing name. Errors only occur when an array has `size=0`.